man/systemd.netdev.xml

```diff
@@ -819,6 +819,13 @@
           <para>Accepts the same key in [VXLAN] section.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Independent=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true vxlan does not require .network file. Defaults to <literal>false</literal>.
```

Comment:
I find this confusing, i.e. the primary effect of this is that the netdev is not associated with an underlying iface, and that's what should be mentioned here, no?